API/Data/Migrations/20210630155028_DefaultDarkMode.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```

Comment:
Migration wasn't created, so we can rollback the migration and thus remove these files. 

By defaulting it on the model, it will ensure when the first user is created, the settings has dark mode enabled.

Reply:
Removed migrations.

Marking resolved.